src/utils/ordinal.ts

```diff
@@ -0,0 +1,16 @@
+/**
+ * Returns number param as ordinal.
+ * Source: https://stackoverflow.com/a/31615643/101290
+ *
+ * @param n number
+ *
+ * @return ordinal string
```

Comment:
Rather than `@param` and `@return`, consider something like this:

```
@example

  ordinal(1)  // '1st'
  ordinal(42) // '42nd'
  ordinal(99) // '99th'
```

Reply:
👍🏼